chatbot_production/chatbot_service.py

```
@@ -0,0 +1,76 @@
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+INTENTS_PATH = os.path.join(BASE_DIR, "intents.json")
+
+# Download NLTK resources (once)
+nltk.download("punkt")
+nltk.download("wordnet")
+
+# Load model and data
+with open(INTENTS_PATH, "r", encoding="utf-8") as file:
+    intents = json.load(file)
+
+words = pickle.load(open("words.pkl", "rb"))
```

Ruff (0.13.1)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)